### ISSUE TITLE
Adding the dependency causes build to fail

### ISSUE THREAD

**Issue by zedlabs** [mikepenz/MaterialDrawer#2481]
## About this issue

- added the dependency implementation "com.mikepenz:materialdrawer:7.0.0-rc05" to my build.gradle and the build failed, it starts to build fine if I remove the dependency
This is the error message - `More than one file was found with OS independent path 'META-INF/library-core_release.kotlin_module'`

## Details
-  Used library version - 7.0.0-rc05
- Used support library version - v4:1.0.0 
- Used gradle version - 5.4.1
-Used gradle plugin version - 3.5.0
-Other used libraries, potential conflicting libraries
```
def room_version = "2.2.0-alpha02"
    def nav_dep = "2.2.0-alpha01"
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
    implementation 'androidx.appcompat:appcompat:1.0.2'
    implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
    testImplementation 'junit:junit:4.12'
    implementation "androidx.annotation:annotation:1.1.0"
    androidTestImplementation 'androidx.test:runner:1.2.0'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.2.0'
    //recyclerView
    implementation 'androidx.recyclerview:recyclerview:1.0.0'
    //glide
    implementation 'com.github.bumptech.glide:glide:4.9.0'
    annotationProcessor 'com.github.bumptech.glide:compiler:4.9.0'
    //retrofit
    implementation 'com.squareup.retrofit2:retrofit:2.6.0'
    implementation 'com.squareup.retrofit2:converter-gson:2.6.0'
    implementation 'com.squareup.okhttp3:logging-interceptor:4.0.1'
    //kotlin coroutines
    //implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.0-RC'
    //viewModel and liveData
    implementation "androidx.lifecycle:lifecycle-extensions:2.0.0"
    //Room
    implementation "androidx.room:room-runtime:$room_version"
    kapt "androidx.room:room-compiler:$room_version"
    implementation "androidx.room:room-ktx:$room_version"
    //Paging
    implementation "androidx.paging:paging-runtime:2.1.0"
    //Material
    implementation "com.google.android.material:material:1.1.0-alpha09"
    // Navigation
    implementation "androidx.navigation:navigation-fragment-ktx:$nav_dep"
    implementation "androidx.navigation:navigation-ui-ktx:$nav_dep"
    //photoView
    implementation 'com.github.chrisbanes:PhotoView:2.3.0'
```

**Comment by zedlabs** [comment]
adding the following to the android{ } in the build.gradle file fixed it, but it seems like a bad fix:
```
packagingOptions {
        exclude 'META-INF/library-core_release.kotlin_module'
    }
```

**Comment by mikepenz** [comment]
@zedlabs I had the same with various google libraries. will investigate more, on how to get rid of it

**Comment by Rokomor** [comment]
Seems that both libraries (dependencies in MaterialDrawer):
_Iconics_ and _FastAdapter_ have this file `META-INF/library-core_release.kotlin_module`
that's why Task :mergeDebugJavaResource fails
@zedlabs suggestion helps to exclude one of them

**Comment by mikepenz** [comment]
fixed in v7.0.0-rc06
